spec/parallel_tests/cli_spec.rb

```diff
@@ -149,6 +149,36 @@ def call(*args)
       end
     end
   end
+
+  describe "#display_duration" do
+
+    def call(*args)
```

Comment:
fwiw, I don't like naming a method call() unless I'm explicitly overriding ruby's Method#call.  Naming something else call() can potentially ruin my day if I come along later and do functional programming tricks, instrumentation tools, or other dark arts.

But when in Rome... ;)